pom.xml

```diff
@@ -267,6 +267,7 @@
                         <exclude>**/*.id</exclude>
                         <exclude>**/*.gz</exclude>
                         <exclude>.editorconfig</exclude>
+                        <exclude>.github/dependabot.yml</exclude>
```

Comment:
I think is better to put a license header in top of the file.

Reply:
I don't have a strong opinion either way, but I would argue that it is not straight forward to claim copyright for automatically generated files.